.circleci/PR_SITE_NUM.sh

```diff
@@ -0,0 +1,3 @@
+PR_NUM="$(echo "$CIRCLE_PULL_REQUEST" | perl -ne "/(\d+)\$/; print \$1")"
+PR_SITE_NUM="$(expr $PR_NUM % $PR_SITE_COUNT)"
```

Comment:
From Codacy:
> Double quote to prevent globbing and word splitting.

This is important for strings that could have spaces but would be confusing for variables where spaces should be impossible.

> PR_SITE_NUM appears unused. Verify use (or export if used externally).

This file is expected to be `source`ed, so the variable becomes part of the sourcing script's environment. It is up to that script to decide whether an `export` makes sense.